src/Schema/Expect.php

```diff
@@ -24,8 +24,8 @@
  * @method static Type float($default = null)
  * @method static Type bool($default = null)
  * @method static Type null()
- * @method static Type array()
- * @method static Type list()
+ * @method static Type array($default = [])
+ * @method static Type list($default = [])
  * @method static Type mixed()
```

Comment:
hmmm, for `mixed` too

Reply:
Btw, it should be noted in docs that default value for array and list is `[]`, not `null`

> (by default, all properties are optional and have default value null)